packages/react-liveness/src/components/FaceLivenessDetector/service/utils/constants.ts

```diff
@@ -7,6 +7,16 @@ export const FACE_DISTANCE_THRESHOLD = 0.32;
 export const REDUCED_THRESHOLD = 0.4;
 export const REDUCED_THRESHOLD_MOBILE = 0.37;
 
+// Constants from science team to determine ocular distance (space between eyes)
+export const ALPHA = 2.0;
+export const GAMMA = 1.8;
+
+// Constants from science team to find face match percentage
+export const RANGE_MIN = 0;
+export const RANGE_MAX = 1;
+export const WEIGHT_MIN = 0.25;
+export const WEIGHT_MAX = 0.75;
```

Comment:
Can we give these constants more descriptive names? For example, `ALPHA` doesn't provide a ton of context in to what it correlates to

Reply:
Updated ALPHA and GAMMA, will double check with science team because I'm not super clear on what they mean either. Was it just those or all of them?